common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -87,13 +88,13 @@ public enum Operation {
     }
 
     @Getter
-    @NonNull final private Operation operation;
```

Comment:
make this change because getter somehow doesn't play nice with lombok here (And we shouldn't need it to begin with)

Intentionally keep the change minimum so that i don't bloat up the PR and scare people off.